<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInterface</class>
 <widget class="QDialog" name="DialogInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Joint</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonJointColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of JOINT class elements</string>
        </property>
        <property name="text">
         <string>Joint</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditJointX"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditJointY"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditJointZ"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Geom</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonGeomColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of GEOM class elements</string>
        </property>
        <property name="text">
         <string>Geom</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGeomX"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditGeomY"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditGeomZ"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonGeomForceColour">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of GEOM force indicator</string>
        </property>
        <property name="text">
         <string>Force</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditGeomForceRadius">
        <property name="toolTip">
         <string>Radius of force indicator</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditGeomForceScale">
        <property name="toolTip">
         <string>Scale of force indicator</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Body</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonBodyColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of BODY class elements</string>
        </property>
        <property name="text">
         <string>Body</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditBodyX"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditBodyY"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditBodyZ"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Environment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonEnvironmentColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of ENVIRONMENT class elements</string>
        </property>
        <property name="text">
         <string>Environment</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditEnvironmentY"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditEnvironmentZ"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditEnvironmentX"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonBackgroundColor">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Reporter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonReporterColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of JOINT class elements</string>
        </property>
        <property name="text">
         <string>Reporter</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditReporterX"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditReporterY"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditReporterZ"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonDataTargetColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDataTargetX"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditDataTargetY"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditDataTargetZ"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Muscle</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonStrapColour">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of MUSCLE class elements straps</string>
        </property>
        <property name="text">
         <string>Muscle</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditStrapRadius">
        <property name="toolTip">
         <string>Muscle strap radius</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonStrapCylinderColour">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of MUSCLE class elements cylinders</string>
        </property>
        <property name="text">
         <string>Cylinder</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditStrapCylinderLength">
        <property name="toolTip">
         <string>Length to draw muscle wrap cylinders</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonStrapForceColour">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Set the colour of MUSCLE force indicators</string>
        </property>
        <property name="text">
         <string>Force</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditStrapForceRadius">
        <property name="toolTip">
         <string>Radius of muscle force indicator</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEditStrapForceScale">
        <property name="toolTip">
         <string>Scale value of muscle force indicator</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogInterface</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogInterface</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
